includes/blocks/safe-svg/block.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://schemas.wp.org/trunk/block.json",
-  "apiVersion": 2,
+  "apiVersion": 3,
```

Comment:
@fabiankaegy I approved this pull request but i want to confirm whether it will work fine with WordPress `5.7` (plugin minimum dependency).

Reply:
@fabiankaegy bumping this back up for your input

Reply:
So based on the [documentation](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-api-versions/), v3 of the API is WP 6.3+, whereas v2 is 5.6+. That said, not clear from that if things won't work at all or if it gracefully falls back to an older API version.

Just tested in a WP 5.7 environment and the SVG block doesn't work, whether using v1, v2 or v3. Existing blocks still render but adding a new SVG block throws an error.

Updated to 5.8 and things worked, both on v2 and seemingly on v3 as well. Not a super thorough test but I think we should look to bump our minimum WordPress version to at least 5.8 since it doesn't seem the block works on 5.7 anyway.

Still defer to @fabiankaegy's opinion on if there's any concern with using API v3 on an environment running a WP version less than 6.3. If no concern, I'd suggest we bump the WP version in this PR. Otherwise that can be done in a separate PR and maybe we close this out until 6.3 is our minimum

Reply:
I'm fine bumping the WP minimum to 6.1 for that matter, been considering a L-2 support policy for WP so if we bump "tested up to" 6.3 then that would be ok by me.